internal/resources/metal/organization_member/resource_test.go

```diff
@@ -104,28 +94,45 @@ resource "equinix_metal_project" "test" {
 }
 
 func testAccResourceMetalOrganizationMember_owner() string {
-	return fmt.Sprintf(`
+	return `
 	resource "equinix_metal_organization_member" "owner" {
-		invitee = "/* TODO: Add org owner email or token owner email here */"
+		invitee = "tfacc.testing.member@equinixmetal.com"
 		roles = ["owner"]
-		projects_ids = []
+		project_ids = []
```

Comment:
Tying in to Oscar's comment elsewhere, the name of this attribute shouldn't change (it's surprising to me that the upgrade tests didn't have trouble with this, since changing an attribute name is a breaking change that would impact how the resource is stored in state).
```suggestion
		projects_ids = []
```

Reply:
@codinja1188 the PR view still shows the old code.  Could you push your updates?